playbooks/openstack/configuration.md

```diff
@@ -777,6 +777,29 @@ enable_kuryr_cni_probes: True
 enable_kuryr_cni_probes: True
 ```
 
+### kuryr-controller in HA mode
+
+Since Rocky version of Kuryr-Kubernetes it is possible to run
+kuryr-controller in Active/Passive HA mode. This means that each
+kuryr-controller pod will run additional leader-elector container that
+will be responsible for providing it with current leader name.
+
+To enable HA you need to set:
```

Comment:
HA is enable by default even if the number of replicas is 1, perhaps worth to rephrase it as: to change the number of replicas...